account_asset_management/models/account_asset.py

```diff
@@ -279,6 +279,8 @@ class AccountAsset(models.Model):
         readonly=False,
         store=True,
     )
+    editable_lines = fields.Boolean(string="Allow editing the depreciation board")
```

Comment:
I assume, that the first one only allows edition and the second one is for adding lines, isn't it?

Reply:
The first one, is to completely bypass the automitic/computed method so one could use a custom board.